diff --git a/drivers/gpu/drm/panel/msm89x7-generated/panel-xiaomi-ugglite-otm1901a-tm.c b/drivers/gpu/drm/panel/msm89x7-generated/panel-xiaomi-ugglite-otm1901a-tm.c
index 6c2681ddbdf5..7d25da35a691 100644
--- a/drivers/gpu/drm/panel/msm89x7-generated/panel-xiaomi-ugglite-otm1901a-tm.c
+++ b/drivers/gpu/drm/panel/msm89x7-generated/panel-xiaomi-ugglite-otm1901a-tm.c
@@ -22,7 +22,7 @@ struct tm_otm1901a {
        struct mipi_dsi_device *dsi;
        struct regulator_bulk_data *supplies;
        struct gpio_desc *reset_gpio;
-       struct gpio_desc *backlight_gpio;
+       // struct gpio_desc *backlight_gpio;
 };

 static const struct regulator_bulk_data tm_otm1901a_supplies[] = {
@@ -172,7 +172,7 @@ static int tm_otm1901a_bl_update_status(struct backlight_device *bl)
        u16 brightness = backlight_get_brightness(bl);
        int ret;

-       gpiod_set_value_cansleep(ctx->backlight_gpio, !!brightness);
+       // gpiod_set_value_cansleep(ctx->backlight_gpio, !!brightness);

        dsi->mode_flags &= ~MIPI_DSI_MODE_LPM;

@@ -225,10 +225,10 @@ static int tm_otm1901a_probe(struct mipi_dsi_device *dsi)
                return dev_err_probe(dev, PTR_ERR(ctx->reset_gpio),
                                     "Failed to get reset-gpios\n");

-       ctx->backlight_gpio = devm_gpiod_get(dev, "backlight", GPIOD_OUT_LOW);
-       if (IS_ERR(ctx->backlight_gpio))
-               return dev_err_probe(dev, PTR_ERR(ctx->backlight_gpio),
-                                    "Failed to get backlight-gpios\n");
+       // ctx->backlight_gpio = devm_gpiod_get(dev, "backlight", GPIOD_OUT_LOW);
+       // if (IS_ERR(ctx->backlight_gpio))
+       //      return dev_err_probe(dev, PTR_ERR(ctx->backlight_gpio),
+       //                           "Failed to get backlight-gpios\n");

        ctx->dsi = dsi;
        mipi_dsi_set_drvdata(dsi, ctx);
